### DISCUSSION TITLE
Querying sub-collections/nested collections on Firestore

### DISCUSSION THREAD

**Discussion by irinagabe** [Budibase/budibase#7261] [Help] [answered]
I'm really excited to build my first internal tool on Budibase, but I'm struggling with querying the nested collections on Firestore. There's no documentation on this so far, so any help will be appreciated.

The document structure I'm working with is:
Root Collection
- Sub-Collection 1
- Sub-Collection 2

**Comment by melohagan** [comment, marked as answer]
Hey @irinagabe 

I've added a short section to the docs explaining this: https://docs.budibase.com/docs/firestore#subcollection

**Reply by irinagabe** [reply]
Thanks, does this mean the "docId" can be replaced by bindings? @melohagan

**Reply by melohagan** [reply]
You can do, I've updated the docs: https://docs.budibase.com/docs/firestore#subcollection

This page may also help: https://docs.budibase.com/docs/custom-queries

**Reply by irinagabe** [reply]
This is super helpful, thanks a bunch @melohagan
